axelor-project/src/main/resources/views/Announcement.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<object-views xmlns="http://axelor.com/xml/ns/object-views" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+	xsi:schemaLocation="http://axelor.com/xml/ns/object-views http://axelor.com/xml/ns/object-views/object-views_5.0.xsd">
+  
+  <grid name="announcement-grid" title="Announcement" model="com.axelor.apps.project.db.Announcement">
+    <field name="title"/>
+    <field name="project"/>
+    <field name="content"/>
+    <field name="date"/>
+  </grid>
+  
+  <form name="announcement-form" title="Announcement"
+    model="com.axelor.apps.project.db.Announcement" width="large"
+    onNew="action-project-record-announcement-onnew">
+    <panel>
```

Comment:
Add name for panel. Follow naming convention for panel , means all panel names should be ends with 'Panel'. 
This is single main panel, so its name should be 'mainPane'. Take care of this in future development.